packages/account/src/Components/poi/status/upload-complete/__tests__/upload-complete.spec.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Button } from '@deriv/components';
+import { PlatformContext } from '@deriv/shared';
+import { UploadComplete } from '../upload-complete.jsx';
+
+jest.mock('Components/poa/poa-button', () => jest.fn(() => <div data-testid='poa-button' />));
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Icon: jest.fn(() => <div data-testid='dt_mocked_icon' />),
+    };
+});
+
+describe('<UploadComplete />', () => {
+    const successful_upload_message = /Your proof of identity was submitted successfully/i;
+    const poi_under_review_message = /We’ll review your document and notify you of its status within 1 to 2 hours./i;
+    const redirect_button = <Button>Lorem epsom</Button>;
+    const poa_under_review_message = /Your document is being reviewed, please check back in 1-3 days./i;
+    const needs_poa_extra_submit_message = /You must also submit a proof of address./i;
+
+    const RenderWithTrue = component =>
+        render(<PlatformContext.Provider value={{ is_appstore: true }}>{component}</PlatformContext.Provider>);
+
+    const RenderWithFalse = component =>
+        render(<PlatformContext.Provider value={{ is_appstore: false }}>{component}</PlatformContext.Provider>);
+
+    it('should display Icon if is_appstore is false', () => {
+        RenderWithFalse(<UploadComplete />);
+
+        expect(screen.getByTestId(/dt_mocked_icon/)).toBeInTheDocument();
+    });
+
+    it('should render <UploadComplete /> component and is_appstore is true', () => {
+        RenderWithTrue(<UploadComplete />);
+
+        expect(screen.getByText(successful_upload_message)).toBeInTheDocument();
+        expect(screen.getByText(poi_under_review_message)).toBeInTheDocument();
+        expect(screen.getByTestId(/dt_mocked_icon/i)).toBeInTheDocument();
+    });
+
+    it('should not show redirect_button if it is not passed', () => {
+        RenderWithTrue(<UploadComplete />);
+
```

Comment:
maybe we can check this case in previous test case, because there is no changed props